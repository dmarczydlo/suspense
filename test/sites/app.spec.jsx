import React from 'react';
import App from '../../src/sites/app';

describe('app site', () => {
    let wrapper = null;

    it('should be rendered', () => {
        wrapper = global.shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});
