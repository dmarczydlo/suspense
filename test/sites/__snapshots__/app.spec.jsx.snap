// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app site should be rendered 1`] = `
<div>
  <RouterElements />
</div>
`;
